src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -338,14 +352,21 @@ export class GlobalMap extends React.Component<Props, State> {
     const { activeShows } = this.state
     const hasShows = activeShows.length > 0
 
+    // Check if it's an iPhone with ears (iPhone X, Xr, Xs, etc...)
+    const iPhoneHasEars = this.props.safeAreaInsets.top === 44
```

Comment:
Should we `=== 44` or do something like `> 20`? What if the ears get bigger, for example?

Reply:
yeah > 20 makes sense. Because it'll be `=== 20` on everything else